ci/conda_env_archery.yml

```diff
@@ -15,8 +15,28 @@
 # specific language governing permissions and limitations
 # under the License.
 
+# cli
 click
-gitpython
+
+# bot, crossbow
+github3.py
+jinja2
+jira
+pygit2
 pygithub
 ruamel.yaml
+setuptools_scm
+toolz
```

Comment:
Are you sure about all these? We should only list direct dependencies here.

Reply:
I derived this list from [setup.py](https://github.com/apache/arrow/blob/master/dev/archery/setup.py#L27-L34)